# this array is used to save execution start/end time of navigator task and a preemption flag 

uint32[6] time		# timestamps for navigator task and a preemption flag 


